### PR TITLE
Add Enterprise Management service to project

### PR DESCRIPTION
**Pull request by padamstx**
## PR summary
This PR adds the Enterprise Management service to the project, including the generated
service and unit test code, plus a skeleton integration test.

**Fixes:**

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -6,9 +6,9 @@ dist/
 node_modules/
 examples/**/node_modules/
 
-test/resources/auth.js
+test/resources/*.js
 # ignore emitted js
 **/*v*.js
-!test/**/*.js
+!test/integration/*.js
```
Comment (dpopp07) [inline .eslintignore:12]: What's the reason for this change? I think we do want to lint the unit tests

**Comment by padamstx** [comment]
@mahakAg I've updated the add-em branch with latest from master, etc.  There are test failures in the integration test for EM.  Could you please take a look and see if you can fix those, then push changes to the add-em branch (to be included in this PR)?
